packages/manager/src/features/Linodes/LinodesDetail/LinodeSettings/LinodeSettingsAlertsPanel.tsx

```diff
@@ -112,7 +116,11 @@ export const LinodeSettingsAlertsPanel = (props: Props) => {
       onStateChange: (
         e: React.ChangeEvent<HTMLInputElement>,
         checked: boolean
-      ) => formik.setFieldValue('io', checked ? 10000 : 0),
+      ) =>
+        formik.setFieldValue(
+          'io',
+          checked ? (linode?.alerts.io ? linode?.alerts.io : 10000) : 0
```

Comment:
Maybe should just remove the default values? The API will always return something here. We'd just have to check that the linode is not `undefined` first.

```suggestion
          checked ? linode.alerts.io : 0
```

Reply:
This was a good thought. When I tried this for a threshold, such as `checked ? linode?.alerts.io : 0`, we end up in a situation where an alert can be toggled off, but then can't be toggled back on -- since the API returns `0` when an alert is disabled, and the toggle state is enabled when the value > `0`, the toggle seems perpetually disabled. 

https://github.com/linode/manager/assets/114685994/1f86f201-a871-47cc-a20e-509cfc94e375